workflow/rocoto/tasks.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, app_config: AppConfig, cdump: str) -> None:
         # Save dict_configs and base in the internal state (never know where it may be needed)
         self._configs = self.app_config.configs
         self._base = self._configs['base']
+        self._base['minus_cycle_interval'] = '-{h:02n}:00:00'.format(h=self._base["assim_freq"])
```

Comment:
One should not define a specific use variable so high in the class.
```suggestion
        self._base['cycle_interval'] =  to_timedelta(f"{self._base['assim_freq']}H") 
```